src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DeliverySessionServiceTest.kt

```diff
@@ -196,7 +205,38 @@ class DeliverySessionServiceTest @Autowired constructor(
       assertThat(error.message).contains("can't schedule new appointment for session; latest appointment has no feedback delivered")
     }
   }
+  @Test
+  fun `superseded flag set to false when pop attended appointment`() {
+    val actionPlan = actionPlanFactory.createApproved(numberOfSessions = 1)
+    val session = deliverySessionFactory.createScheduled(attended = Attended.YES, referral = actionPlan.referral)
+
+    val deliverySession = session.currentAppointment?.attended?.let {
+      deliverySessionService.recordAppointmentAttendance(
+        defaultUser, actionPlan.id, session.sessionNumber,
+        it, ""
+      )
+    }
+
+    val appointment = deliverySession?.appointments?.first()
+
+    assertThat(appointment?.superseded).isFalse
+  }
+  @Test
+  fun `superseded flag set to true when pop did not attended appointment`() {
```

Comment:
Can we replace the acronym with the expanded version, please? It would help future developers a lot